app/views/layouts/application.html.slim

```diff
@@ -2,7 +2,10 @@ doctype html
 html
   head
     title Portus
-    = stylesheet_link_tag 'application', media: 'all'
+    - if Rails.application.assets.find_asset(params[:controller])
+      = stylesheet_link_tag params[:controller], media: 'all'
+    - else
+      = stylesheet_link_tag 'application', media: 'all'
```

Comment:
In most cases the asset loaded is the `application.css` and all controller assets needs to import `application.scss` because of some variables and classes defined there. So, I thought that will be better add this `if` than repeat only this import in a "empty" file (and for each controller).